platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java

```diff
@@ -815,7 +816,7 @@ public final void animateCamera(final CameraUpdate update, final int durationMs,
   /**
    * Invalidates the current camera position by reconstructing it from mbgl
    */
-  void invalidateCameraPosition() {
```

Comment:
why do u declare invaldateCameraPosition twice?

Reply:
This has been refactored since this commit landed and is not the case anymore.